src/test/HelmetTest.js

```diff
@@ -217,7 +217,20 @@ describe("Helmet", () => {
 
                 const htmlTag = document.getElementsByTagName("html")[0];
 
-                expect(htmlTag.attributes.length).to.equal(0);
+                expect(htmlTag.getAttribute("lang")).to.be.null;
+                expect(htmlTag.getAttribute("amp")).to.be.null;
```

Comment:
@cwelch5 how is that different from case 1? Because helmet **will** override/clear attributes once they have been present in helmet. Unless there's another use case I'm not thinking about